api/src/resources/comment.rs

```diff
@@ -565,6 +588,13 @@ pub struct PredictedLabel {
     pub probability: NotNan<f64>,
 }
 
+#[derive(Debug, Clone, Deserialize, Serialize, PartialEq, Eq)]
+pub struct AutoThresholdLabel {
+    pub name: Vec<String>,
+    pub probability: NotNan<f64>,
+    pub auto_thresholds: Vec<String>,
```

Comment:
Isn't that a bunch of numbers instead?

Reply:
No, it looks like this (coming from the Predict api routes):

```
    {
      "labels": [
        {
          "name": [
            "Margin Call"
          ],
          "probability": 0.9999445676803589,
          "auto_thresholds": [
            "high_recall",
            "balanced",
            "high_precision",
            "sampled_0",
            "sampled_1",
            "sampled_2"
          ]
        },
        {
          "name": [
            "Margin Call",
            "Confirmation"
          ],
          "probability": 0.9999445676803589,
          "auto_thresholds": [
            "high_recall",
            "balanced",
            "high_precision",
            "sampled_0",
            "sampled_1",
            "sampled_2"
          ]
        }
      ]
    }
```
i.e. it shows which thresholds a given predicted label meets.

Reply:
Oh, didn't realise! I had forgotten how this worked.